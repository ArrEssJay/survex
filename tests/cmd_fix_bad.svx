; pos=fail error=10
*fix A 0
*fix B 0 0
*fix C 0 0 0 1.0 2.0 3.0 3.14
*fix D 0 0 0 1.0 2.0 3.0 3.14 2.18
*fix E test

*fix Ar reference 0
*fix Br reference 0 0
*fix Cr reference 0 0 0 1.0 2.0 3.0 3.14
*fix Dr reference 0 0 0 1.0 2.0 3.0 3.14 2.18
*fix Er reference test

; so we actually have some survey data
*fix 1 reference 0 0 0
