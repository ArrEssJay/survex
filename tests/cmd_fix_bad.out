./cmd_fix_bad.svx:1:9: Expecting numeric field, found ""
./cmd_fix_bad.svx:2:11: Expecting numeric field, found ""
./cmd_fix_bad.svx:3:30: Expecting numeric field, found ""
./cmd_fix_bad.svx:4:35: Expecting numeric field, found ""
./cmd_fix_bad.svx:5:12: Expecting numeric field, found "test"
./cmd_fix_bad.svx:7:20: Expecting numeric field, found ""
./cmd_fix_bad.svx:8:22: Expecting numeric field, found ""
./cmd_fix_bad.svx:9:41: Expecting numeric field, found ""
./cmd_fix_bad.svx:10:46: Expecting numeric field, found ""
./cmd_fix_bad.svx:11:23: Expecting numeric field, found "test"

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 0 warning(s) and 10 error(s) - no output files produced.
